code/ai/aigoals.cpp

```diff
@@ -1995,7 +1995,7 @@ ai_achievability ai_mission_goal_achievable( int objnum, ai_goal *aigp )
 			// short circuit a couple of cases.  Ship not arrived shouldn't happen.  Ship gone means
 			// we mark the goal as not achievable.
 			if ( status == SHIP_STATUS_NOT_ARRIVED ) {
-				Int3();										// get Allender.  this shouldn't happen!!!
+				UNREACHABLE("get Allender.  this shouldn't happen!!!");	// get Allender.  this shouldn't happen!!!
```

Comment:
This message needs more descriptive info.  You can keep get Allender, but someone who doesn't have the skill or desire to look through our source code needs to know the severity of the mistake and what they should do about it.